### ISSUE TITLE
Iterate a folder for the metadata selection field

### ISSUE THREAD

**Issue by senner** [collective/wildcard.pfg.stripe#1]
On the edit Stripe field on the new select widget for metadata to send, it won't iterate a "Folder" of fields, only the parent.

**Comment by vangheem** [comment]
For instance, fieldset folders.

**Comment by vangheem** [comment]
There might be a magic method on the PFG object to automatically get all the fields on the form regardless of where there are(folders, etc) too.

**Comment by davisagli** [comment]
FWIW, fieldset folders are deprecated in PloneFormGen because there were a lot of bugs similar to this one. Instead you can add fieldset start/end markers within the main form folder, and keep a flat list of fields.

**Comment by vangheem** [comment]
@davisagli yes, I know :) We're just trying to give @jdmartens a nice easy noob-friendly plone task to work on :)

**Comment by davisagli** [comment]
Then by all means, carry on! :+1:
